src/components/VirtualList/Config.ts

```diff
@@ -0,0 +1,109 @@
+import deepFreeze from 'deep-freeze';
+import { AppListItem } from '../../types/AppList';
+import {
+  ApplicationsIcon,
+  AutomationIcon,
+  BundleIcon,
+  CodeBranchIcon,
+  GlobeRouteIcon,
+  IntegrationIcon,
+  LockedIcon,
+  MigrationIcon,
```

Comment:
I'm not sure what's wrong with this icon (might be in patternfly) but it triggers a warning in console: `Warning: Invalid DOM property 'class'. Did you mean 'className'?
    in svg (created by MigrationIcon)`